cli/install/install.go

```diff
@@ -0,0 +1,654 @@
+}
+
+// checkDependecie checks if programm is installed.
+func checkDependecie(programm string) bool {
+	cmd := exec.Command(programm, "--version")
+	cmd.Start()
+	err := cmd.Wait()
+	return err == nil
+}
+
+// checkPythonDependencie checks if python packet is installed.
+func checkPythonDependencie(packet string) bool {
+	cmd := exec.Command("pip3", "list")
+	readPipe, writePipe, _ := os.Pipe()
+	cmd.Stdout = writePipe
+	cmd.Stderr = os.Stderr
+	cmd.Start()
+	cmd.Wait()
+	writePipe.Close()
+	var buf bytes.Buffer
+	io.Copy(&buf, readPipe)
+	packets := buf.String()
+	return strings.Contains(packets, packet)
+}
+
+// checkDeianLibDependencie checks if library is installed on Debian/Ubuntu.
+func checkDebianLibDependencie(library string) bool {
+	cmd := exec.Command("dpkg", "-L", library)
+	cmd.Start()
+	if cmd.Wait() == nil {
+		return true
+	} else {
+		return false
+	}
+}
+
+// checkLibDependencie checks if library is installed
+func checkLibDependencie(library string) bool {
+	cmd := exec.Command("")
+	if strings.Contains(detectOsName(), "Ubuntu") || strings.Contains(detectOsName(), "Debian") {
+		return checkDebianLibDependencie(library)
+	}
+	if strings.Contains(detectOsName(), "darwin") {
+		cmd = exec.Command("brew", "list")
+	}
+	if strings.Contains(detectOsName(), "CentOS") {
+		cmd = exec.Command("yum", "list", "--installed")
+	}
+	readPipe, writePipe, _ := os.Pipe()
+	cmd.Stdout = writePipe
+	cmd.Stderr = os.Stderr
+	cmd.Start()
+	cmd.Wait()
+	writePipe.Close()
+	var buf bytes.Buffer
+	io.Copy(&buf, readPipe)
+	packets := buf.String()
+	return strings.Contains(packets, library)
+}
+
+// checkTtDependencies checks if tt dependencies is installed.
+func checkTtDependencies() bool {
+	programms := []string{"mage", "git", "python3"}
+	var miss_count = 0
+	missing_pack := ""
+	for i := 0; i < len(programms); i++ {
+		if !checkDependecie(programms[i]) {
+			miss_count++
+			missing_pack += " " + programms[i]
+		}
+	}
+	osName := detectOsName()
+	if miss_count != 0 {
+		log.Error("The operation requires some dependencies.")
+		log.Error("Missing packages: " + missing_pack)
+		if osName == "darwin" {
+			log.Error("You can install them by running commands:")
+			log.Error("brew install" + missing_pack)
+		} else if strings.Contains(osName, "CentOs") {
+			log.Error("You can install them by running command:\n sudo yum install" + missing_pack)
+		} else if strings.Contains(osName, "Ubuntu") || strings.Contains(osName, "Debian") {
+			log.Error("You can install them by running command:\n sudo apt install" + missing_pack)
+		}
+		log.Error("Usage: tt install -f if You already have those packages installed")
+		return false
+	}
+	return true
+}
+
+// checkTarantoolDependencies checks if tarantool dependencies is installed.
+func checkTarantoolDependencies() bool {
+	var miss_count = 0
+	missing_pack := ""
+	missing_pack_python := ""
+	// Tarantool dependencies.
+	programms := []Packet{{"cmake", "cmake"}, {"git", "git"}, {"python3", "python3"},
```

Comment:
Are these packages required by tarantool or by specific command? Pls, rename a function accordingly.

Reply:
It is list of Tarantool dependencies from Tarantool.io